PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -117,8 +125,8 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   /// \param collision collision to test against the selection criteria
   /// \param centrality collision centrality variable to be set in this function
   /// \return bitmask with the event selection criteria not satisfied by the collision
-  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll>
-  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality)
+  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll, typename BC>
+  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, const BC& bc, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb)
```

Comment:
```suggestion
  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, const BC& bc, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb)
```
I don't think it is needed to pass `bc` as argument. It can be retrieved from `collision`.

Reply:
Hmm, I get your point, but I can also see that you had to add almost 30 new lines `auto bc = ` just to pass `bc` as the argument.

Reply:
Done in [f5be35a](https://github.com/AliceO2Group/O2Physics/pull/6533/commits/f5be35a8d879a866908f2e13c4eb38408c6b05ae). I am still not convinced that accessing twice per candidate the same table is worth sparing 3 lines of code per task, but given that is minor if you prefer so we can proceed like this. Thanks!

Reply:
If any, I think it's a negligilbe overhead I would say, so we can proceed with the new one 👍